src/qt/walletmodel.cpp

```diff
@@ -561,9 +561,9 @@ bool WalletModel::getPubKey(const CKeyID &address, CPubKey& vchPubKeyOut) const
     return wallet->GetPubKey(address, vchPubKeyOut);
 }
 
-bool WalletModel::havePrivKey(const CKeyID &address) const
+isminetype WalletModel::isMine(const CTxDestination &dest) const
```

Comment:
function names are PascalCase.

Reply:
Check `src/qt/walletmodel.h` - they're all lowerThenUpper

Reply:
Yes but it's outdated.

Reply:
So what's the plan? Shouldn't we go through the code fixing all the outdated style at once rather than fixing one at a time?

Reply:
I mean, I can change just the one I touched if that's preferable.